samples/ballerina-by-example/examples/content-based-routing-service/content-based-routing-service.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina.net.http;
+import ballerina.lang.jsons;
+import ballerina.lang.messages;
+import ballerina.doc;
+
+@doc:Description {value : "Service is invoke using BasePath value (/cbr)."}
+@http:BasePath {value:"/cbr"}
+service contentBasedRouting {
+ @doc:Description {value : "http:POST{} annotation declares the HTTP method."}
+ @http:POST{}
+ @http:Path {value:"/route"}
+ resource cbrResource (message m) {
+   //Create service endpoint using HTTP client-connector.
+   http:ClientConnector locationEP = create
+              http:ClientConnector("http://www.mocky.io");
+   //Declare a string within inverted commas
```

Comment:
We don't need to explain on how to define a string etc. It will be explained in the 'value type' example...